package.json

```diff
@@ -41,7 +41,6 @@
     "typescript": "2.3.0"
   },
   "scripts": {
-    "postinstall": "grunt shell:webdriverUpdate",
```

Comment:
The error says `"Failed at the protractor-sync@1.0.4 postinstall script 'grunt update'"` but this is `grunt shell:webdriverUpdate`. Is there a reference to an `update` task hiding somewhere else?